uttut/pipeline/ops/pure_num_to_token.py

```diff
@@ -0,0 +1,69 @@
+import re
+from typing import List, Tuple
+
+from .base import Operator, NullLabelAligner
+from ..edit import lst2lst
+from ..edit.replacement import ReplacementGroup
+from .tokens import NUM_TOKEN
+
+
+class PureNum2Token(Operator):
+
+    _input_type = list
+    _output_type = list
+
+    REGEX_PATTERN = re.compile(r"[\d\uFF10-\uFF19]+")
+
+    def __init__(self, token: str = None):
+        """
+
+        Recognize string in a list and replace it with a special token (_num_)
```

Comment:
Given a list of strings, replace strings which have only numeric characters with a special token.

Reply:
f1c5e47